### PR TITLE
Set version to 2.2.0

### PR DESCRIPTION
**Pull request by jvegasbsc**
Just what it says

-   [ ] [☝ Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do

## Checklist

-   [x] PR has a descriptive title for the [changelog](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#branches-pull-requests-and-code-review)
-   [x] Code follows the [style guide](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#code-style)
-   [x] [Documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#documentation) is available for new functionality
-   [x] YAML files pass [`pre-commit`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#pre-commit) or [`yamllint`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/community/introduction.html#yaml) checks
-   [ ] [Circle/CI tests pass](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#branches-pull-requests-and-code-review)
-   [x] [Codacy code quality checks pass](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#branches-pull-requests-and-code-review)
-   [x] [Documentation builds successfully](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#branches-pull-requests-and-code-review) on readthedocs
-   [x] [Unit tests](https://docs.esmvaltool.org/projects/esmvalcore/projects/esmvalcore/en/latest/contributing.html#contributing-to-the-esmvalcore-package) are available

If you make backwards incompatible changes to the recipe format:

-   [ ] Update [ESMValTool](https://github.com/esmvalgroup/esmvaltool) and link the pull request(s) in the description

***

To help with the number pull requests:

-   🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValTool/pulls) in this repository

### CODE REVIEW

**Comment by valeriupredoi** [comment]
good one @jvegasbsc - we're on the road again :grin: A few things:

- after we merge this, we need the release branch - ask @mattiarighi to set no-push/no-merge permissions on that - or you have these prerogatives already, if you do pls set them on that branch, just you and me can work on it; then we'll ask peeps to run recipes; we should also make sure that you have the pypi and anaconda credentials to upload the packages via automatic Github Actions - maybe we can chat over a call soon about all this jazz?

For here:
- you'll have to change the version in `esmvalcore/_version.py`  and in `package/meta.yaml` too
- you'll have to change the `date-released: 2020-10-12` in `CITATION.cff` too

:beer:

**Comment by jvegasbsc** [comment]
Yes, tomorrow morning it's ok? I can not set an hour now, but will contact you in the morning

**Comment by valeriupredoi** [comment]
sounds good, my morning starts at 11am your time, so no rush :grin:

**Comment by jvegasbsc** [comment]
I updated the zenodo link to version 2.1, so at least it does not point to a beta release and set up the release date to next monday. We may need to update it, but its a better approximation just in case we forget.

By the way, I love this unique identifier loop: I need to do the release to get the number but I would like to have the number in the release metadata

File: CITATION.cff
Comment (bouweandela) [inline CITATION.cff:162]: This DOI should not be changed, it's a generic DOI for all versions.

**Review by bouweandela — CHANGES_REQUESTED** [review]
Did you find the instructions we have available for making a release? They live here: https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#how-to-make-a-release

One of the things mentioned there is the list of files in which the version number lives. It looks like you forgot to update `package/meta.yaml`, could you update the version number in that file too?

File: CITATION.cff
Comment (jvegasbsc) [reply CITATION.cff:162]: If you go there it says that there is a newer version available. That's why I proposo to update it

Comment (bouweandela) [reply CITATION.cff:162]: It doesn't for me, could it be your browser cache or something? This is the DOI listed on zenodo as the generic DOI for all versions:
> Cite all versions? You can cite all versions by using the DOI [10.5281/zenodo.3387139](https://doi.org/10.5281/zenodo.3387139). This DOI represents all versions, and will always resolve to the latest one. [Read more](http://help.zenodo.org/#versioning).

Comment (jvegasbsc) [reply CITATION.cff:162]: A Firefox + Google issue: firefox does not resolve the DOI url and searchs google, which in turn presents as the first result not the actual generic DOI but a specific version one

Comment (jvegasbsc) [reply CITATION.cff:162]: I am reverting the change

**Comment by valeriupredoi** [comment]
let's have a list of things that we want merged in the release branch before release https://github.com/ESMValGroup/ESMValCore/issues/982

**Review by bouweandela — APPROVED** [review]

**Review by valeriupredoi — APPROVED** [review]

**Comment by valeriupredoi** [comment]
this is beautiful: 
buffg

**Pull request merged**
